skimage/util/tests/test_montage.py

```diff
@@ -21,15 +21,16 @@ def test_montage_simple_gray():
     assert_array_equal(arr_out, arr_ref)
 
 
-def test_montage_simple_rgb():
+def test_montage_simple_rgb(channel_pos=-1):
```

Comment:
not sure what `channel_pos` is used for here

Reply:
It should not be there. I originally started modifying this test class, but later decided to leave this one using `multichannel` and add a new `test_montage_simple_rgb_channel_axes` test case instead to test the `channel_axis` argument.